numba/tests/test_range.py

```diff
@@ -177,6 +187,31 @@ def test_range_attrs(self):
         for arg in arglist:
             self.assertEqual(cfunc(*arg), pyfunc(*arg))
 
+    @tag('important')
+    def test_range_contains(self):
+        pyfunc = range_contains
+        arglist = [(0, 0, 1),
```

Comment:
You may want to check all permutations of `0, 1, -1` to make sure everything is covered. Also, you may want to consider looking at max values too to check not only around 0 and 1 but also at the 'fringes' of the number line.

Reply:
I added all of the permutations mentioned. I wasn't sure about the need for the max value checks, so I left them out for now.